lib/apple_sign_in/helpers/jwt_conditions.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: false
+
+require 'byebug'
+
+module AppleSignIn
+  class JWTConditions
+    include Conditions
+
+    CONDITIONS = [
```

Comment:
I was so abstract because i thought that write the code in this way was better than write
JWT::ClaimsValidator.new(decoded_jwt).validate! && valid_sub? && valid_aud? && valid_exp? && valid_iss? && valid_iat? && .. etc

Reply:
ok, it's just a matter of style not a blocker to merge the pr